docs/user-guide/share.md

```diff
@@ -1,49 +1,82 @@
+* Directly from MapStore [Homepage](home-page.md) by clicking on the Share button <img src="../img/button/share.jpg" class="ms-docbutton"/> present in each resource card toolbar
 
 <img src="../img/share/share_cards.jpg" class="ms-docimage"  style="max-width:400px;"/>
 
-* Inside the resource (Map, Dashboard etc) by clicking on the [Burger Menu](menu-bar.md#burger-menu) <img src="../img/button/sharebm.jpg" class="ms-docbutton" style="max-height:25px;"/> option (only for Maps)
+* Inside the resource by selecting the <img src="../img/button/share-button-bm.jpg" class="ms-docbutton" style="max-height:25px;"/> option from [Burger Menu](menu-bar.md#burger-menu) 
 
-The user is allowed to save a resource in different ways:
+From the *Share panel* the user is allowed to save a resource in different ways:
 
-* With a direct link
+* With a **direct link**
 
-* Through a social network
+* Through a **social network**
 
-* With embedded code or APIs
+* With **embedded code** or **APIs**
 
 ## Link
 
-As soon as the Share window opens, the link section is by default displayed:
+As soon as the *Share panel* opens, the link section is by default displayed:
 
 <img src="../img/share/share_window.jpg" class="ms-docimage"  style="max-width:400px;"/>
 
-Here it is possible to copy the resource url link and access its qr code. Moreover only opening this window from the [Burger Menu](menu-bar.md#burger-menu) it is possible to access the advanced options where the user can add the bbox param or center and zoom param of the share link.
-
-<img src="../img/share/share_window_center_zoom.jpg" class="ms-docimage"  style="max-width:400px;"/>
-
-With *Add center and zoom to sharing link*, manually or by clicking the center coordinates with possibility to switch coordinate format. The user can also set *zoom* value from the input field. By default, the center and zoom param are added to the share link with the possibility to add a place marker (when map loads) by selecting the *Add Marker* checkbox. 
+Here the user can copy the resource **URL link** and passed it to a new clipboard or access the shared map through the **QR code**.
 
 ## Social
 
-The social section displays like the following:
+Switch the panel on the social section the user can share the resource on **Facebook**, **Twitter** and **LinkedIn** simply by clicking on the social icon.
 
 <img src="../img/share/social.jpg" class="ms-docimage"  style="max-width:400px;"/>
 
-From here it is possible to share the Resource on Facebook, Twitter and LinkedIn. Also in this case, only opening the window from the maps [Burger Menu](menu-bar.md#burger-menu) the user can add the bbox or center and zoom param.
-
 ## Embed
 
-Accessing the embed section it displays the following: 
+Switch the panel on the embed section the user can copy the **embedded code** or the **APIs**. It is useful, for example, for inserting the resource inside a webpage.
```

Comment:
The **Embed** section provides to the user the needed snippets, **embedded code** or the **MS APIs** to embed MapStore in a third party web page.